website/src/css/custom.scss

```diff
@@ -0,0 +1,26 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;600&display=swap');
+
+@import 'admonitions';
+@import 'benefits-section';
+@import 'footer-socials';
+@import 'footer';
+@import 'header-github';
+@import 'navbar';
+@import 'showcase-link';
+@import 'showcase-section';
+@import 'utilities';
+
+.docusaurus-highlight-code-line {
```

Comment:
This can be extracted to `_markdown.scss`.

Reply:
@noomorph - do you mean lines 14 - 18 and 24 - 26?

Reply:
Looks like it.

As for`[data-theme='light'] .banner {`, I doubt it is used at all.

Reply:
I think it belongs to the UA banner

Reply:
Maybe someone it belongs by intent, but not by any actual effect. See:

<img width="765" alt="image" src="https://user-images.githubusercontent.com/1962469/216572422-431a061c-bfb1-45a7-8b0c-38938fb48be3.png">

The class `banner_SLir` != `banner`, so this rule looks useless.

Reply:
Thanks. I will remove it since it's not doing anything.